### ISSUE TITLE
Assets not linked if not linking dependencies

### ISSUE THREAD

**Issue by olistic** [react-native-community/cli#584]
## Environment

```
React Native Environment Info:
    System:
      OS: macOS 10.14.5
      CPU: (4) x64 Intel(R) Core(TM) i7-7567U CPU @ 3.50GHz
      Memory: 32.57 MB / 16.00 GB
      Shell: 5.7.1 - /usr/local/bin/zsh
    Binaries:
      Node: 12.4.0 - /usr/local/bin/node
      Yarn: 1.16.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5522156
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
```

## Description

For `react-native-cli` version 1.3.0, project assets are not linked when running `react-native link` if not linking dependencies. I think the bug is at [this line](https://github.com/react-native-community/cli/blob/d1db45a386273fa78dbc580859001d65d595bfaa/packages/cli/src/commands/link/linkAll.js#L46). `linkAssets` is called for each dependency config, instead of only once after having collected all the assets.

I managed to link my assets by making this change:

```diff
const tasks = flatten(
    depenendenciesConfig.map(config => [
      () => promisify(config.commands.prelink || commandStub),
      () => linkDependency(platforms, project, config),
      () => promisify(config.commands.postlink || commandStub),
-      () => linkAssets(platforms, project, assets),
    ]),
-  );
+  ).concat([
+    () => linkAssets(platforms, project, assets),
+  ]);
```

I can submit a PR after you guys confirm this is actually a bug and that is a proper fix.

Thanks!

## Reproducible Demo

n/a

**Comment by thymikee** [comment]
You're right, it's fixed in v2, see how it works here: https://github.com/react-native-community/cli/blob/master/packages/cli/src/commands/link/linkAll.js

If you care enough, feel free to send a PR to the `1.x` branch :)

**Comment by olistic** [comment]
@thymikee Thanks for your quick response. I'm stuck with RN 0.59.10 (and thus the v1.3.0 of the CLI, right? 🤔) because I couldn't make 0.60 work on my monorepo yet.

I'm sending the PR shortly!

**Comment by thymikee** [comment]
The 1.x is currently at 1.11.0, here's how to [update the `@react-native-community/cli`](https://github.com/react-native-community/cli#updating-the-cli) to the latest version.

**Comment by thymikee** [comment]
Fixed in 1.11.2
